### PR TITLE
PLT-5634 Add account settings scene

### PR DESCRIPTION
**Pull request by csduarte**
#### Summary
This PR adds the account settings scene. Note that all setting options have been disabled for this screen at this point. Each setting option will be enabled when the corresponding screen is implemented. The following is the finished account settings screen:

simulator screen shot feb 24 2017 7 30 13 am

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-5634

#### Checklist
- [x] Has UI changes

@jarredwitt

### CODE REVIEW

File: app/scenes/account_settings/account_settings.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+        return (
+            <View style={style.wrapper}>
+                <View style={style.container}>
+                    <View style={style.itemsContainer}>
+                        {this.buildItemRow('gear', 'user.settings.modal.general', 'General', () => true, true, false)}
+                        {this.buildItemRow('lock', 'user.settings.modal.security', 'Security', () => true, true, false)}
+                        {this.buildItemRow('bell', 'user.settings.modal.notifications', 'Notifications', () => true, true, false)}
+                        {this.buildItemRow('mobile', 'user.settings.modal.display', 'Display', () => true, true, false)}
+                        {this.buildItemRow('wrench', 'user.settings.modal.advanced', 'Advanced', () => true, false, false)}
```
Comment (enahum) [inline app/scenes/account_settings/account_settings.js]: I wonder if we can do this by having a `renderOptions` method and inside that method we can call the `buildItemRow` function?

**Comment by jarredwitt** [comment]
@enahum changes made.

File: app/scenes/account_settings/account_settings.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            </View>
+        );
+    }
+
+    renderItems = () => {
+        const {theme} = this.props;
+        const style = getStyleSheet(theme);
+
+        return (
+            <View style={style.itemsContainer}>
```
Comment (enahum) [inline app/scenes/account_settings/account_settings.js]: Why is this here? I thought `renderItems` would return an array something like

```js
return [
    this.buildItemRow('gear', 'user.settings.modal.general', 'General', () => true, true, false),
    ...
];
```

**Comment by jarredwitt** [comment]
@enahum added ```key={id}```
